### PR TITLE
Remove unused scripts, fix broken contact form, set up subdomain to uwbacm

### PR DESCRIPTION
**Pull request by mduw**
**UPDATES: #2** 
- Removed google-analytics id. Replace with ACM's GA id, if needed.
- Set up subdomain of uwbacm: [competitiveprog.uwbacm.com](https://competitiveprog.uwbacm.com)
- Added sitemap 
- Added searching keywords in` _config.yml`
- Added breaking-new banner for hot news announcement.
- Added custom 404 page
- Replaced img with license-free images

**UPDATES: #1 **
- Removed unused scripts, files
- Contact form did not work properly in the older version. Google form is embedded into contact form in this version.
- ~~Updated google-analytics id in .yml~~
- Changed size and color of buttons on the navigation bar and change title size

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -54,38 +54,33 @@
 </ul>
 
 
-<form name="sentMessage" id="contactForm" novalidate>
+<form action="https://docs.google.com/forms/u/0/d/e/1FAIpQLSfX2eMW8InjQMwXqHl8Uk1Xk8-GlwJ275CarAFL-0itDGtBwg/formResponse" method="POST" id="mG61Hd">
```
Comment (alanboy) [inline contact.html:57]: wait we can post directly to a google form from this html form? omg thats great.

File: _includes/google-analytics.html [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-170203794-1"></script>
```
Comment (alanboy) [inline _includes/google-analytics.html]: i wonder why the variable did not work. would be preferable to have it as a variable.

File: _config.yml [hunk 3]
```diff
@@ -11,8 +11,8 @@ discord_link:       https://discord.gg/2tWhanq
 github_link:        https://github.com/UWB-ACM
 linkedin_link:      https://www.linkedin.com/in/uwbacm/
 
-# Add your google-analytics ID here to activate google analytics
-google_analytics:   UA-XXXXXXXXX-X # out your google-analytics code
+# Google-analytics ID
```
Comment (alanboy) [inline _config.yml]: can you add a comment here that says that this value is also hard-coded in `google-analytics.html` ?

File: _config.yml [hunk 4]
```diff
@@ -11,8 +11,8 @@ discord_link:       https://discord.gg/2tWhanq
...
+google_analytics:   UA-170203794-1
```
Comment (alanboy) [inline _config.yml]: who owns this GA?

**Review by alanboy — COMMENTED** [review]
Please also update the title of the PR so it reflects a little better the changes you are making.

File: _includes/google-analytics.html (hunk 2 above)
Comment (mduw) [reply _includes/google-analytics.html]: I changed it to variable in update #2

File: _config.yml (hunk 3 above)
Comment (mduw) [reply _config.yml]: fixed!

File: _config.yml (hunk 4 above)
Comment (mduw) [reply _config.yml]: good catch! this is my GA id for testing. I removed it in update #2. will replace with ACM's GA id if needed.

**Comment by mduw** [comment]
Desktop view: 
Screen Shot 2020-07-27 at 1 23 22 AM

**Comment by mduw** [comment]
MObile view:
Screen Shot 2020-07-27 at 1 24 20 AM

**Review by alanboy — APPROVED** [review]

**Pull request merged**